frontend/src/components/messstelle/gesamtauswertung/stepper/AuswertungStepper.vue

```diff
@@ -117,14 +117,19 @@ const selectedTagesTypAsSummary = computed(() => {
   return summary;
 });
 const selectedOrtAsSummary = computed(() => {
-  const mstIds = auswertungOptions.value.mstIds;
+  const messstelleAuswertungIds =
+    auswertungOptions.value.messstelleAuswertungIds;
   let summary = ``;
-  if (mstIds.length > 1) {
-    summary = `Mst-Id's': ${mstIds.join(", ")}`;
+  if (messstelleAuswertungIds.length > 1) {
+    const allMstIds: Array<string> = [];
```

Comment:
Evtl mehr Streamen.

ToArray macht den Code resilienter gegen undefined und null und die Iteration mit Extraktion geschieht in einem Aufruf.

```ts
    const allMstIds: Array<string> = _.toArray(messstelleAuswertungIds).map((messstelleAuswertungId) => messstelleAuswertungId.mstId);


```

Reply:
Das toArray kann evtl auch bereits weiter oben ausgeführt werden.

![image](https://github.com/user-attachments/assets/8f969b9a-4e91-4aff-a840-2c8e21152afb)